service/controller/v3/resource/deployment/resource.go

```diff
@@ -27,31 +26,26 @@ const (
 )
 
 type Config struct {
-	CloudConfig *cloudconfig.CloudConfig
-	Debugger    *debugger.Debugger
-	Logger      micrologger.Logger
+	Debugger *debugger.Debugger
+	Logger   micrologger.Logger
 
-	Azure       setting.Azure
-	AzureConfig client.AzureClientSetConfig
+	Azure           setting.Azure
+	HostAzureConfig client.AzureClientSetConfig
```

Comment:
Is `HostAzureConfig` needed at all in this resource?

Reply:
good point, its not used anymore